apps/chat/src/utils/app/file.ts

```diff
@@ -466,3 +466,11 @@ export const isConversationHasExternalAttachments = (
     return attachmentBucket !== userBucket;
   });
 };
+
+export const validateMimeFormat = (type: string) => {
+  const reg = new RegExp(
+    '^([a-zA-Z0-9!*\\-.+]+|\\*)\\/([a-zA-Z0-9!*\\-.+]+|\\*)$',
+  );
```

Comment:
put RegExp into constant. The issues happen with `test`-method only if we use global flag into RegExp